catalog/video/catalog-mpegts-stream/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -52,6 +52,11 @@
                 name="Location Subsample Count" id="klvLocationSubsampleCount" required="true"
                 type="Integer" default="50"/>
 
+        <AD
+                description="Metacard Update Initial Delay (milliseconds)"
```

Comment:
Can we add a little more to the description here since this will show up as a tool tip in the admin UI?  Short explanation of why we want to delay this.  And if at all possible, don't make an admin convert it to millis.  Take seconds and do the conversion in the code.